osu.Game/Screens/Play/HUD/PlayerSettingsOverlay.cs

```diff
@@ -3,27 +3,21 @@
 
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
-using osu.Framework.Input.Events;
 using osuTK;
 using osu.Game.Screens.Play.PlayerSettings;
-using osuTK.Input;
 
 namespace osu.Game.Screens.Play.HUD
 {
     public partial class PlayerSettingsOverlay : VisibilityContainer
     {
         private const int fade_duration = 200;
 
-        public bool ReplayLoaded;
-
         public readonly PlaybackSettings PlaybackSettings;
 
         public readonly VisualSettings VisualSettings;
 
         public PlayerSettingsOverlay()
         {
-            AlwaysPresent = true;
```

Comment:
should be a bit of a good optimisation

Reply:
`AlwaysPresent` with no inline comment as to why... :grimacing: :grimacing: :grimacing: love these. unconditional `PropagateNonPositionalInputSubTree` too.

both things look safe to remove though. blame is fun on this one, https://github.com/ppy/osu/pull/1336 added the `AlwaysPresent` presumably for input handling but failed to notice problem with presence, which then https://github.com/ppy/osu/pull/2005 fixed without removing presence flag. love 2018 code. good riddance

Reply:
There was a comment there which explains both of them:

https://github.com/ppy/osu/pull/24129/files#diff-94587203ff289de741a04133d63b6024dcc3303a994e4a9e68092fc1104c328cL52